tests/test_class_attributes.rs

```diff
@@ -235,6 +235,26 @@ fn test_renaming_all_struct_fields() {
     });
 }
 
+#[pyclass(get_all, set_all, new = "from_fields")]
+struct AutoNewCls {
+    a: i32,
+    b: String,
+    c: Option<f64>,
+}
+
+#[test]
+fn new_impl() {
+    Python::attach(|py| {
+        // python should be able to do AutoNewCls(1, "two", 3.0)
+        let cls = py.get_type::<AutoNewCls>();
+        pyo3::py_run!(
+            py,
+            cls,
+            "inst = cls(1, 'two', 3.0); assert inst.a == 1; assert inst.b == 'two'; assert inst.c == 3.0"
+        );
+    });
+}
+
```

Comment:
It would be good to add a test with a tuple struct, e.g. 

```rust
struct Point2d(f64, f64);
```

... I would think the generated constructor would allow only positional inputs, as there are no names for the fields.